site/src/sections/author.style.js

```diff
@@ -0,0 +1,248 @@
+import styled from "styled-components";
+
+
+const AuthorStyleWrapper = styled.div`
+    
+    background: radial-gradient(149.45% 344.41% at 102.05% -2.23%, #3C494F 0%, #00B39F 100%);
+    postion: fixed;
+    
+    .background-image {
+        position: fixed;
+        height: 2288px;
+        right: 0px;
+        height: 100%;
+        background: radial-gradient(149.45% 344.41% at 102.05% -2.23%, #3C494F 0%, #00B39F 100%)
+    }
+    
+
+    .author-content{
+        position: absolute;
+        margin-top: 6%;
+
+    }
+    .author-title{
+        text-align: center;
+        color: #ffffff;
+    }
+    .aboutLee{
```

Comment:
it would be nice to have a little more space between the image and about it , see figma to see what i want to say .